build.sc

```diff
@@ -514,7 +514,7 @@ object testrunner extends MillPublishScalaModule {
   def moduleDeps = Seq(scalalib.api, main.util, entrypoint)
 
   object entrypoint extends MillPublishJavaModule {
-    override def runIvyDeps = Agg(Deps.sbtTestInterface)
+    override def ivyDeps = Agg(Deps.sbtTestInterface)
```

Comment:
```suggestion
    override def ivyDeps = super.ivyDeps() ++ Agg(Deps.sbtTestInterface)
```

Reply:
Is there something we miss here. I think the `entrypoint` module is pretty dependency less, except the runtime dependency on `test-interface`.

Reply:
yeah there are no supers here to worry about. And this is probably the right thing: we *want* it to be dependency-free except for what's listed, rather than adding onto whatever may be there before